configs/AARCH64_quartz64_verified.cmake

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env -S cmake -P
+#
+# Copyright 2025, Proofcraft Pty Ltd
+#
+# SPDX-License-Identifier: GPL-2.0-only
+#
+
+include(${CMAKE_CURRENT_LIST_DIR}/include/AARCH64_verified_include.cmake)
+
+set(KernelPlatform "quartz64" CACHE STRING "")
```

Comment:
Is there any cost to adding these additional platforms as "verified" when the platform may not necessarily meet a quality threshold on the practical systems side? eg this Pine64 RK3566 Quartz64-A Board platform isn't on the docsite and I don't think it has enabled hardware tests.

Reply:
My guideline so far has been to add the things that are built in CI, but if it's not even on the supported platforms list, we should probably not include it, even if the proofs pass.

Reply:
Have removed it now.